exercises/ex_atm.py

```diff
@@ -0,0 +1,137 @@
+# ATM Exercise
+
+import sys
+
+FILE_PATH = 1
+PASSWORD_INDEX = 0
+BALANCE_INDEX = 1
+
+
+def read_file_move_to_dict(file_path):
+    """Read file and move atm information to dictionary.
+
+    Args:
+        file_path (string): path to atm information file.
+
+    Return:
+        A dictionary, the key is each users id.
+        the value is a list of user's password and balance.
```

Comment:
In the `Return:` section of the documentation, the doc need to start with the type of the returned value, and then the description of that value.

```suggestion
       dict. With users ID as keys and user's data as values.
```